src/analysis/convex/topology.lean

```diff
@@ -199,11 +199,10 @@ convex_iff_open_segment_subset.mpr $ λ x hx y hy,
 protected lemma convex.closure {s : set E} (hs : convex 𝕜 s) : convex 𝕜 (closure s) :=
 λ x hx y hy a b ha hb hab,
 let f : E → E → E := λ x' y', a • x' + b • y' in
-have hf : continuous (λ p : E × E, f p.1 p.2), from
-  (continuous_fst.const_smul _).add (continuous_snd.const_smul _),
-show f x y ∈ closure s, from
-  mem_closure_of_continuous2 hf hx hy (λ x' hx' y' hy', subset_closure
-  (hs hx' hy' ha hb hab))
+have hf : continuous (function.uncurry f),
+  from (continuous_fst.const_smul _).add (continuous_snd.const_smul _),
+show f x y ∈ closure s,
+  from map_mem_closure₂ hf hx hy (λ x' hx' y' hy', hs hx' hy' ha hb hab)
```

Comment:
This makes it look like `mem_closure_of_continuous2` was a stronger lemma as it didn't need as strong an assumption. Is my reading correct?

Presumably the difference in strength has no mathematical content, so

bors d+

Reply:
Indeed, these lemmas differ by `closure_closure` and all applications of `mem_closure_of_continuous2` did not use this extra generality.